docs/index.rst

```diff
@@ -3,15 +3,91 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-ReX documentation
+ReX: Causal Responsibility Explanations for image classifiers
 =================
 
-Add your content using ``reStructuredText`` syntax. See the
-`reStructuredText <https://www.sphinx-doc.org/en/master/usage/restructuredtext/index.html>`_
-documentation for details.
+**ReX** is a causal explainability tool for image classifiers.
+ReX is black-box, that is, agnostic to the internal structure of the classifier.
+We assume that we can modify the inputs and send them to the classifier, observing the output.
+ReX outperforms other tools on single explanations, non-contiguous explanations (for partially obscured images), and multiple explanations.
 
+.. image:: ../assets/rex-structure-600x129.png
+  :width: 600
+  :alt: ReX organisation
+
+For more information and links to the papers, see the :doc:`background` page.
+
+Installation
```

Comment:
The installation and quickstart sections look to be the same as in the README, apart from `.md`/`.rst` formatting. Could you import them from one file into the other? Maybe something like [this](https://muffinresearch.co.uk/selectively-including-parts-readme-rst-in-your-docs/) could work?

Reply:
Nice find - I knew it was possible to imclude a file, I didn't know you could include only part of a file! I think I got this working now.